CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Improved typography in quotes - thanks @apollisa
 - Simplifying add todo interface
+- Toggle AM / PM for 24 hour time
```

Comment:
Is this feature for 24 hour time or 12 hour time?

Reply:
:man_facepalming: I got that the wrong way round.